src/main/java/net/minestom/arena/group/Group.java

```diff
@@ -10,6 +12,9 @@ static Group findGroup(@NotNull Player player) {
         return GroupManager.getGroup(player);
     }
 
+    void play(Arena arena);
```

Comment:
Very misleading as this does not do anything. Should be internal.

Reply:
Fixed in latest commit